lighthouse-core/audits/accessibility/aria-allowed-attr.js

```diff
@@ -33,6 +33,8 @@ class ARIAAllowedAttr extends AxeAudit {
       category: 'Accessibility',
       name: 'aria-allowed-attr',
       description: 'Element aria-* attributes are allowed for this role',
+      helpText: 'Invalid role-attribute combinations can disable the accessibility ' +
```

Comment:
Suggest rewording to something like "Each ARIA `role` defines a specific subset of `aria-*` attributes that it supports. Mismatching these will invalidate the aria-* attributes."